src/aboutsettings.h

```diff
@@ -61,21 +61,6 @@ class SYSTEMSETTINGS_EXPORT AboutSettings: public QObject
     explicit AboutSettings(QObject *parent = 0);
     virtual ~AboutSettings();
 
-    // Deprecated -- use diskUsageModel() instead
-    Q_INVOKABLE qlonglong totalDiskSpace() const;
-    // Deprecated -- use diskUsageModel() instead
-    Q_INVOKABLE qlonglong availableDiskSpace() const;
-
-    /**
-     * Returns a list of JS objects with the following keys:
-     *  - storageType: one of "mass" (mass storage), "system" (system storage) or "user" (user storage)
-     *  - path: filesystem path (e.g. "/" or "/home/")
-     *  - available: available bytes on the storage
-     *  - total: total bytes on the storage
-     **/
-    Q_INVOKABLE QVariant diskUsageModel() const;
-    Q_INVOKABLE void refreshStorageModels();
```

Comment:
Could update src/plugin/plugins.qmltypes to drop these removed methods, I think.

Reply:
Sure, though as internal api keeping those up to date isn't that mandatory. There's some manually executed tooling side to eventually get these synced.

Reply:
:man_shrugging:

Reply:
Done